packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -1149,15 +1150,13 @@ function makeAugmentedSchema(
                 [connectionField.fieldName]: {
                     type: connection.NonNull,
                     args: composeNodeArgs,
-                    resolve: (source, args: ConnectionQueryArgs) => {
-                        const { totalCount: count, edges } = source[connectionField.fieldName];
-
-                        const totalCount = isInt(count) ? count.toNumber() : count;
-
-                        return {
-                            totalCount,
-                            ...createConnectionWithEdgeProperties(edges, args, totalCount),
-                        };
+                    resolve: (rootValue, args: ConnectionQueryArgs, ctx, resolveInfo: GraphQLResolveInfo) => {
```

Comment:
Can we change `rootValue` back to `source`? We're quite inconsistent with this in our resolver function signatures, and [`source` is the name used in the official codebase](https://github.com/graphql/graphql-js/issues/2842#issuecomment-725499214).

Ideally the same for `ctx` to `context` and `resolveInfo` to `info` - they're all only stylistic changes, but we should be aiming to have a consistent codebase.

Reply:
I must say I do find this odd https://github.com/graphql/graphql-js/blob/c589c3d285cb1ec44b09bf0b50ec041ec083760c/src/graphql.ts#L62